poly-synth/index.html

```diff
@@ -55,7 +56,11 @@
     </ul>
     <p>
     The combined output of the voices can then be modulated by bitcrushing,
-    reverb, and delay effects.
+    reverb, and delay effects. If noise gate side chaining is active, then
+    the output of the synthesizer will ramp towards silence when the level of
+    the drum sample exceeds the specified threshold. The beat and its volume is
```

Comment:
No, the drum interrupts the sustained note. When the volume of the drum exceeds the threshold, the synthesizer will map towards silence because we subtract the weight from 1 `outputChannel0[j] = (1 - weights[j]) * signalChannel0[j];`